spec/prog/vm/nexus_spec.rb

```diff
@@ -14,7 +14,7 @@
   let(:vm) { Vm.new(size: "m5a.2x").tap { _1.id = "a410a91a-dc31-4119-9094-3c6a1fb49601" } }
   let(:tg) { TagSpace.create(name: "default").tap { _1.associate_with_tag_space(_1) } }
 
-  describe "#assemble" do
+  describe ".assemble" do
```

Comment:
Is this kind of a new convention that we want assemble function tests to start with `.` and states with`#`? If so, makes sense. If not and some kind of an extra change you didn't intend to do, I still like it 👍

Reply:
Oh yeah I looked it up, it is the convention for class methods, but it also was an extra change.